test/export/InstanceExporter.test.ts

```diff
@@ -2945,13 +2953,21 @@ describe('InstanceExporter', () => {
       // Parent: Patient
       // * identifier ^slicing.discriminator.type = #value
       // * identifier ^slicing.discriminator.path = "value"
+      // * identifier ^slicing.rules = #open
       // * identifier contains ParentSlice 1..1
       // * identifier[ParentSlice] ^slicing.discriminator.type = #value
       // * identifier[ParentSlice] ^slicing.discriminator.path = "value"
+      // * identifier ^slicing.rules = #open
```

Comment:
Since this is with the other `ParentSlice` slicing rules, I think this comment is supposed to be:
```suggestion
      // * identifier[ParentSlice] ^slicing.rules = #open
```